src/CounterInterceptor/Program.cs

```diff
@@ -0,0 +1,84 @@
+﻿namespace TemporalioSamples.CounterInterceptor;
+
+using Temporalio.Client;
+using Temporalio.Worker;
+
+internal class Program
+{
+    private static async Task Main(string[] args)
+    {
+        var client = await TemporalClient.ConnectAsync(
+            options: new("localhost:7233")
+            {
+                Interceptors = new[]
+                {
+                    new SimpleClientCallsInterceptor(),
+                },
+            });
+
+        using var tokenSource = new CancellationTokenSource();
+        Console.CancelKeyPress += (_, eventArgs) =>

```

Comment:
This program is a run then complete, you don't need ctrl+c support as if it was a long running worker. I would recommend just removing cancel token and shutting down the worker manually if you must have the entire program self-contained (i.e. combining client and worker).

Reply:
Bump, this was never addressed

Reply:
Bump again, this is still not addressed

Reply:
Fixed.